Earthfile

```diff
@@ -818,3 +818,28 @@ npm-update-all:
         RUN cd $nodepath && npm update
         SAVE ARTIFACT --if-exists $nodepath/package-lock.json AS LOCAL $nodepath/package-lock.json
     END
+
+# merge-main-to-docs merges the main branch into docs-0.7
+merge-main-to-docs:
+    FROM alpine/git
+    ARG git_repo="earthly/earthly"
+    ARG git_url="git@github.com:$git_repo"
+    ARG to_branch="docs-0.7"
+    ARG from_branch="main"
+    DO github.com/earthly/lib/ssh:ido-add-known_hosts_udc+ADD_KNOWN_HOSTS --target_file=~/.ssh/known_hosts
```

Comment:
I'll change the branch name (`ido-add-known_hosts_udc`) to a (new) tag after merging the branch and publishing a new release in earthly/lib

Reply:
using the commit hash would be even better, since tags can be changed (deleted and created over)